airbyte-db/lib/src/main/java/io/airbyte/db/check/impl/ConfigsDatabaseMigrationCheck.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.check.impl;
+
+import io.airbyte.db.check.DatabaseMigrationCheck;
+import java.util.Optional;
+import org.flywaydb.core.Flyway;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Implementation of the {@link DatabaseMigrationCheck} for the Configurations database.
+ */
+public class ConfigsDatabaseMigrationCheck implements DatabaseMigrationCheck {
```

Comment:
I have the same comment about leveraging lombok to make this class less verbose. Let me know if this would break the compatibility with micronaut.